src/main/groovy/info/solidsoft/gradle/pitest/report/PitestReportAggregator.groovy

```diff
@@ -0,0 +1,111 @@
+package info.solidsoft.gradle.pitest.report
+
+import groovy.transform.CompileStatic
+import info.solidsoft.gradle.pitest.PitestPlugin
+import info.solidsoft.gradle.pitest.PitestPluginExtension
+import info.solidsoft.gradle.pitest.PitestTask
+import org.gradle.api.Project
+import org.gradle.api.UnknownDomainObjectException
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.file.RegularFile
+import org.gradle.api.provider.Provider
+import org.pitest.aggregate.ReportAggregator
+import org.pitest.mutationtest.config.DirectoryResultOutputStrategy
+import org.pitest.mutationtest.config.UndatedReportDirCreationStrategy
+
+import java.nio.file.Paths
+
+@CompileStatic
+class PitestReportAggregator {
+
+    private static final String MUTATION_FILE_NAME = "mutations.xml";
+    private static final String LINECOVERAGE_FILE_NAME = "linecoverage.xml";
+
+    void aggregate(Project root) {
+        Set<Project> subprojects = getSubprojectsWithPitest(root)
+        boolean rootContainPitest = isRootPitestConfigured(root)
```

Comment:
```suggestion
        boolean rootContainPitest = isRootPitestConfigured(root)
        Set<Project> subprojects = getSubprojectsWithPitest(root)
```
I would swap it to be consistent with the order of the next calls.

Btw, `rootContainPitest` → `rootContainsPitest`?